Screens/FavouritePokemonScreen.tsx

```diff
@@ -24,10 +38,58 @@ export const FavouritePokemon = () => {
           alignItems: "center",
         }}
       >
-        <FullImageOfPokemon name={route.params.pokemon} />
-        <Text fontSize="lg" fontFamily="Cochin" color="black" fontWeight="bold">
-          {route.params.pokemon}
+        <ImageOfPokemon name={name} isFullImage={fullImage} />
```

Comment:
W JSX zamiast pisać `prop={true}` wystarczy napisać samo `prop`:
```suggestion
        <ImageOfPokemon name={name} isFullImage />
```